sce.py

```diff
@@ -44,4 +61,5 @@
     elif args.command == 'run':
         handler = SceServiceHandler(args.service, args.dbpath)
```

Comment:
we dont seem to change directories to the cli dir before running things, maybe we need an [`os.chdir`](https://www.tutorialspoint.com/python3/os_chdir.htm) here?